README.md

```diff
@@ -3,6 +3,17 @@ These are packages for using Intel RealSense cameras (D400 and L500 series, SR30
 
 LibRealSense supported version: v2.45.0 (see [realsense2_camera release notes](https://github.com/IntelRealSense/realsense-ros/releases))
 
+## Kiwi additions
+The following additions were made to the wrapper.
+
+#### Custom services
+We added the package [realsense2_camera_srvs](https://github.com/kiwicampus/realsense-ros/tree/feature/realsense-coords-srv/realsense2_camera_srvs) that contains the following services:
```

Comment:
you should use the relative path, i.e. `realsense2_camera_srvs/` because the link you use is specific for this branch that when merged is going to be deleted